packages/app-admin-ui/client/components/CreateItemModal.js

```diff
@@ -73,8 +73,8 @@ class CreateItemModal extends Component {
       variables: { data },
     })
       .then(data => {
-        this.props.onCreate(data);
         this.setState({ item: this.props.list.getInitialItemData({}) });
+        this.props.onCreate(data);
```

Comment:
Re-ordered in the event of `onCreate` throwing?

Reply:
It throws if state is unmounted due to onCreate switch to details page.

I just realize that in case of onCreate fail it will remove data.

 I will use mounted tracking in did mount and willunmount

Reply:
fixed in https://github.com/keystonejs/keystone-5/pull/1724/commits/6229f2ec0b5846f815c1885e1064589dd90af54f

https://github.com/keystonejs/keystone-5/pull/1724/files#diff-eb736ac55cd7d6087c188e1f92167b7dR82-R84

Reply:
Is this issue\comment resolved? If so, can you resolve the conversation.